frontend/src/components/annotation/toolbar.vue

```diff
@@ -1,282 +1,278 @@
 <template>
-<v-toolbar
-      v-if="show_toolbar"
-      dense
-      width="100%"
-      elevation="0"
-      fixed
-      :height="height"
-      style="overflow: hidden; padding:0; border-bottom: 1px solid #e0e0e0"
+  <v-toolbar
```

Comment:
For this file. It seems that mostly what changed was indents right? I think we can have a consensus on how do we set our editor to auto indent code so that we don't have issues where files seems to have changes and there are only indents. We can define it with something like: https://editorconfig.org/

Reply:
Yeah, that totally has sense. We don't have any config file so far, right?